resources/js/Pages/Settings/Role.vue

```diff
@@ -18,139 +20,67 @@ function handleSave() {
 </script>
 
 <template>
-  <Head title="Settings"></Head>
-  <body>
-    <div class="bg-gray-50 text-black/50 dark:bg-black dark:text-white/50">
-      <div class="relative min-h-screen flex flex-col items-center justify-center selection:bg-[#285aff] selection:text-white">
-        <div class="relative w-full max-w-2xl px-6 lg:max-w-7xl">
+  <Head title="Role Settings"></Head>
+  <body class="bg-gray-500 text-black dark:bg-gray-900 dark:text-white">
+    <div class="min-h-screen flex flex-col items-center justify-center">
+      <div class="w-full max-w-6xl p-6">
+
+        <!-- Top Navigation Bar using DaisyUI Navbar -->
+        <div class="navbar bg-gray-800 text-white rounded-lg mb-6 py-4 px-6">
+          <div class="flex-1">
+            <h1 class="text-2xl">Server name</h1>
+          </div>
 
-          <!-- Setting menu -->
-          <div class="split left">
-            <header class="text-center mt-20">
-              <div id="navigation" class="flex flex-col lg:justify-center lg:col-start-2">
-                <h1 class="text1">Server name</h1>
-                <a href="/server-settings" class="rounded-sm hover:text-white hover:ml-10 hover:mr-10 focus:outline-none focus-visible:ring-1 focus-visible:ring-[#285aff] dark:hover:text-white dark:focus-visible:ring-[#285aff] text3">
-                  Server
-                </a>
-                <a href="/role-settings" class="rounded-sm hover:text-white hover:ml-10 hover:mr-10 focus:outline-none focus-visible:ring-1 focus-visible:ring-[#285aff] dark:hover:text-white dark:focus-visible:ring-[#285aff] text3">
-                  Roles
-                </a>
-              </div>
-            </header>
+          <!-- Buttons positioned towards the middle-right with hover expanding effect -->
+          <div class="flex justify-end space-x-6 mr-16">
+            <Link href="/settings/server" class="text-lg transition-all duration-300 ease-in-out hover:bg-gray-700 hover:pl-6 hover:pr-6 p-2 rounded-lg">
+              Server
+            </Link>
+            <Link href="/settings/role" class="text-lg transition-all duration-300 ease-in-out hover:bg-gray-700 hover:pl-6 hover:pr-6 p-2 rounded-lg">
+              Roles
+            </Link>
           </div>
+        </div>
 
-          <!-- Main (Settings) -->
-          <div class="split right">
-            <div class="absolute top-0 right-0 m-4 flex space-x-2">
-              <PrimaryButton>
-                Save Changes
-              </PrimaryButton>
-              <button @click="handleCancel" class="border border-white text-white px-4 py-2 rounded-full hover:bg-red-700 hover:border-red-700 hover:text-white focus:outline-none transition-colors duration-300 ease-in-out">
-                X
-              </button>
-            </div>
+        <!-- Save/Cancel Buttons -->
+        <div class="flex justify-end mb-6 space-x-4">
+          <PrimaryButton @click="handleSave" class="px-6">Save Changes</PrimaryButton>
```

Comment:
better use daisy ui button, just add extra style if its special button, like all caps

Reply:
@M7PAX Better question why do we have does component at all(Primary button, Danger button)?